lib/helpers/export-results.js

```diff
@@ -0,0 +1,83 @@
+
+const fs = require('fs');
+
+/**
+ * @param {Object} opts
+ * @param {String} opts.destFolder  - folder with annotated files
+ * @param {String} opts.resultsFilepath  - a path where should be saved result-file
+ * @returns {} result-file is saved
+ */
+
+/**
+ * @typedef {String} epoch - timestamp, shows when result was generated
+ * @example "2019-11-06T16:32:27.712Z"
+ */
+
+/**
+ * @typedef {Array.<String>} filenames - a list of filenames
```

Comment:
new defined types must start with an uppercase

Reply:
If you are redefining String-type, i suggest you define `Filename` type here, and then you will use `Array.<Filename>`

Reply:
Defining `Filenames` type would be possible but i don't think it's relevant